src/components/Datatable/BatchModule/BatchActions/BatchActions.tsx

```diff
@@ -77,7 +76,7 @@ const BatchActions: FC<BatchActionsProps> = ({ actions }) => {
               <BatchActionButton
                 name={action.name}
                 variant={ButtonVariants.text}
-                onClick={action.onClick}
```

Comment:
was this removed on purpose ?

Reply:
<img width="1932" alt="Screenshot 2023-06-06 at 17 55 37" src="https://github.com/securityscorecard/design-system/assets/18654425/8b8e81e1-969e-4c3a-a0a5-8bf628e00280">

Reply:
I reworked the component so we can leave the `onClick` there 👍